poetry/core/_vendor/tomlkit/_utils.py

```diff
@@ -10,10 +10,7 @@
 from ._compat import timezone
 
 
-try:
-    from collections.abc import Mapping
-except ImportError:
-    from collections import Mapping
+from collections.abc import Mapping
```

Comment:
i assume we don't want to make any changes to the vendored code in `poetry/core/_vendor`

Reply:
ah rats. good catch.
reverted